frappe/core/doctype/system_settings/system_settings.js

```diff
@@ -32,10 +32,25 @@ frappe.ui.form.on("System Settings", {
 			frm.set_value("bypass_restrict_ip_check_if_2fa_enabled", 0);
 		}
 	},
-	on_update: function (frm) {
-		if (frappe.boot.time_zone && frappe.boot.time_zone.system !== frm.doc.time_zone) {
-			// Clear cache after saving to refresh the values of boot.
-			frappe.ui.toolbar.clear_cache();
```

Comment:
Why remove this? 

It does more than just reload tab.

Reply:
Is it because user.py already clears this cache?

Reply:
@ankush: It’s been a while since I created this PR…

But from my recollection – and please correct me if I’m wrong – it’s basically because I found this code never ever runs, as our JS framework doesn‘t even have an `on_update` hook (possibly: not anymore).

If the cache needs to be and isn’t already cleared, we may have to add the line to the `after_save` section. But as 1.) the line hasn’t been executed at all since 576efed7f5a9cd29d01e746bcd506bde4292a811, and 2.) everything seems to work fine without it, I didn’t bother with it.

I will however re-investigate it tomorrow on my dev machine.